daemon/stats_unix_test.go

```diff
@@ -3,28 +3,24 @@
 package daemon
 
 import (
-	"os"
-	"path/filepath"
+	"embed"
+	"strings"
 	"testing"
 
 	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
 )
 
-func TestGetSystemCPUUsageParsing(t *testing.T) {
-	dummyFilePath := filepath.Join("testdata", "stat")
-	expectedCpuUsage := uint64(65647090000000)
-	expectedCpuNum := uint32(128)
-
-	origStatPath := procStatPath
-	procStatPath = dummyFilePath
-	defer func() { procStatPath = origStatPath }()
+//go:embed testdata/stat
+var statData embed.FS
```

Comment:
You can read it directly to a string:
```suggestion
//go:embed testdata/stat
var statData string
```